cmds/ps/ps_linux.go

```diff
@@ -231,8 +232,7 @@ func (pT *ProcessTable) LoadTable() error {
 		if pf.Match([]byte(fi.Name())) {
 			p := &Process{}
 			if err := p.Parse(fi.Name()); err != nil {
-				log.Print(err)
-				return err
+				parsingError = err
 			}
 			pT.table = append(pT.table, *p)
```

Comment:
If there is a parsing error or the process, did you still want to append the process to pT.table?
And why not make pT.table an array of *Process, so you avoid the allocator costs? The dereference (*p) means you are making a copy, which seems unnecessary.